ansible/roles/kube_prometheus_stack/defaults/main/main.yml

```diff
@@ -0,0 +1,342 @@
+---
+
+# The chart to use
+kube_prometheus_stack_chart_repo: https://prometheus-community.github.io/helm-charts
+kube_prometheus_stack_chart_name: kube-prometheus-stack
+kube_prometheus_stack_chart_version: 59.1.0
+
+# Release information
+kube_prometheus_stack_release_namespace: monitoring-system
+kube_prometheus_stack_release_name: kube-prometheus-stack
+
+# The timeout to wait for the release to become ready
+kube_prometheus_stack_wait_timeout: 5m
+
+login_ip: "{{ hostvars[groups['openondemand'][0]]['ansible_host'] }}" # probably needs to be more robust
+control_ip: "{{ ansible_default_ipv4.address| default(ansible_all_ipv4_addresses[0]) }}"
+control_sslip: "{{ control_ip | regex_replace('\\.', '-') }}.sslip.io" 
+
+grafana_volume_size: 10Gi
```

Comment:
Lets try removing the grafana volume entirely. I'm not convinced this would be ok IF k3s was to start grafana after a rebuild, but at the moment it won't.

Reply:
To be clear, giving grafana a volume is optional and is only used for writing data, not storing grafana's entire state (same case for prometheus). Them actually launching seems to be agnostic of any pre-existing storage volumes